python/add_user_to_pulse_subscription.py

```diff
@@ -0,0 +1,262 @@
+# ==========================================================================================================
+# Script Name: Tableau Pulse Metric Subscription Script
+# Author: alaviron@salesforce
+# Date: 10/24/2024
+# Description:
+#   This script subscribes a user to a Pulse metric on Tableau Cloud Site.
+#   The script utilizes the Tableau Server Client (TSC) and requests libraries
```

Comment:
Hm, it shouldn't use TSC. If it does then it shouldn't be writing it's own make_tableau_request methods etc.

Or perhaps it could be moved to the tsc repo and used as a sample there?